.eslintrc.js

```diff
@@ -45,7 +45,33 @@ module.exports = {
   ignorePatterns: ['*.md'],
   overrides: [
     {
-      files: ['layouts/**/*.{js,ts,vue}', 'pages/**/*.vue'],
+      files: [
+        'layouts/**/*.{js,ts,vue}',
+        'pages/**/*.vue',
+        'components/**/*.{js,ts,vue}',
+      ],
+      rules: {
+        'no-restricted-imports': [
+          'warn',
```

Comment:
Cool. Can we force it as an 'error'? Haha 🤣

Reply:
Thanks, I have updated the rule to an 'error'.

The linter will now  'ELIFECYCLE  Command failed with exit code 1".